mod1/src/file-2.xoo

```diff
@@ -0,0 +1,7 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// violate HasTag rule: xoo on line 7
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:41267/organizations/org5/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:41267/project/issues?id=sample&pullRequest=6922&issues=AWlYjnx9m6Dk7Kvw1DWy&open=AWlYjnx9m6Dk7Kvw1DWy)

[ISSUE_KEY]: AWlYjnx9m6Dk7Kvw1DWy